tools/tracegen/Port.cpp

```diff
@@ -684,7 +684,7 @@ Port::omrfile_stat(const char *path, unsigned int flags, struct J9FileStat *buf)
 	if (statfs(path, &statfsbuf)) {
 		return RC_FAILED;
 	}
-	switch (statfsbuf.f_type) {
+	switch ((unsigned long)(statfsbuf.f_type)) {
```

Comment:
As per https://man7.org/linux/man-pages/man2/statfs.2.html, `f_type` is of type `__fsword_t`: 
> The __fsword_t type used for various fields in the statfs
       structure definition is a glibc internal type, not intended for
       public use.  This leaves the programmer in a bit of a conundrum
       when trying to copy or compare these fields to local variables in
       a program.  Using **unsigned int** for such variables suffices on
       most systems.